### ISSUE TITLE
feature(ci): add to the option to skip CI checking

### ISSUE THREAD

**Issue by takeutak** [hyperledger/cactus#1014]
**Is your feature request related to a problem? Please describe.**
According to the current configuration, every time a Pull request is merged into the main branch, the CI check must be redone. Considering that it takes more than an hour to complete a CI check, I think that I cannot cope with a situation where the code needs frequent changes.
However, it is not large enough that all code has to be CI checked every time (For example, there is no need to do CI check on pull-requests including only document files, right?). Since CI check is mandatory for all pull-requests, this inconvenience will occur.

**Describe the solution you'd like**
I would like to be able to choose to avoid CI checks when merging pull-requests. Is it possible to change the specification? @petermetz

**Comment by takeutak** [comment]
I think it can be solved by doing the opposite of the change made in this pull-request #979 . I think this is good, but what do you think? @petermetz

**Comment by RafaelAPB** [comment]
Can our plan of the CI run several PRs concurrently?

**Comment by petermetz** [comment]
@takeutak Agreed. We could add a check in the ci.sh script that examines the diff between `main` and the PR branch, iterates over the files in the diff and checks if it's only .md files then it immediately exists with success.

@RafaelAPB We have the technical capability for that, but at the end of the day we must always execute the tests sequentially to avoid inadvertently merging in bugs that only manifest when two separate PRs are both merged together on main (happened a few times before so that's why I had to configure it in a way that GH always forces you to run the tests against a freshly rebased version of the PR branch)
